components/wb-data-json/data-json-doc-en.html

```diff
@@ -24,7 +24,7 @@ <h2>Use when</h2>
 <section>
 	<h2>Do not use when</h2>
 	<ul>
-		<li>To insert block of HTML content, instead use <a href="../data-ajax/data-ajax-en.html">data-ajax</a> with the filtering option if neccessary.</li>
+		<li>To insert block of HTML content, instead use <a href="https://wet-boew.github.io/themes-dist/GCWeb/demos/data-ajax/data-ajax-en.html">data-ajax</a> with the filtering option if neccessary.</li>
```

Comment:
This URL will need to updated for "gcweb-compiled-demo" repository instead of themes-dist.